gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/PickerPopup.kt

```diff
@@ -42,12 +41,14 @@ import com.composables.core.rememberDialogState
 internal fun PickerPopup(
     anchorAlignment: Alignment.Horizontal,
     anchorBounds: Rect,
+    maxBounds: Rect? = null,
```

Comment:
💡 Although it's very obvious what this name is about now, I'm not sure I would feel the same way sometime in the future. What do you think about renaming to something like `popupDrawArea` or similar that would more cleary describe what is this used for?